play_by_play_test.py

```diff
@@ -0,0 +1,88 @@
+                                 index_col=0,
+                                 dtype=dtypes)
+        self.row_2 = pd.read_csv(directory + '/row_2.csv',
+                                 index_col=0,
+                                 dtype=dtypes)
+        self.row_3 = pd.read_csv(directory + '/row_3.csv',
+                                 index_col=0,
+                                 dtype=dtypes)
+        self.row_4 = pd.read_csv(directory + '/row_4.csv',
+                                 index_col=0,
+                                 dtype=dtypes)
+        self.row_5 = pd.read_csv(directory + '/row_5.csv',
+                                 index_col=0,
+                                 dtype=dtypes)
+        self.row_6 = pd.read_csv(directory + '/row_6.csv',
+                                 index_col=0,
+                                 dtype=dtypes)
+        dataframes = [self.row_1, self.row_2, self.row_3, self.row_4, self.row_5, self.row_6]
+        self.playbyplay = PlayByPlay(dataframes)
+        self.test_data_0 = pd.read_csv(directory + '/test_play_by_play_data.csv',
+                                       index_col=0,
+                                       dtype=dtypes)
+        self.test_data_2 = pd.read_csv(directory + '/test_data_zero_player_id_null.csv',
+                                       index_col=0,
+                                       dtype=dtypes)
+        self.test_data_3 = pd.read_csv(directory + '/test_data_move_team_id.csv',
+                                       index_col=0,
+                                       dtype=dtypes)
+        self.test_data_4 = pd.read_csv(directory + '/test_data_null_empty_player_name.csv',
+                                       index_col=0,
+                                       dtype=dtypes)
+        self.test_data_5 = pd.read_csv(directory + '/test_play_by_play_data_final.csv',
+                                       index_col=0)
+        self.test_data_5['GAME_ID'] = self.test_data_5['GAME_ID'].astype('Int64')
+        self.test_data_5['SCOREMARGIN'] = self.test_data_5['SCOREMARGIN'].astype('Int64')
+        self.test_data_5['PLAYER1_ID'] = self.test_data_5['PLAYER1_ID'].astype('Int64')
+        self.test_data_5['PLAYER2_ID'] = self.test_data_5['PLAYER2_ID'].astype('Int64')
+        self.test_data_5['PLAYER3_ID'] = self.test_data_5['PLAYER3_ID'].astype('Int64')
+        self.test_data_5['PLAYER1_TEAM_ID'] = self.test_data_5['PLAYER1_TEAM_ID'].astype('Int64')
+        self.test_data_5['PLAYER2_TEAM_ID'] = self.test_data_5['PLAYER2_TEAM_ID'].astype('Int64')
+        self.test_data_5['PLAYER3_TEAM_ID'] = self.test_data_5['PLAYER3_TEAM_ID'].astype('Int64')
+        self.test_data_5.columns = self.test_data_5.columns.str.lower()
+
+    def test_build_raw_dataframe(self):
+        result = self.playbyplay._build_raw_dataframe()
+        assert_frame_equal(result, self.test_data_0)
+        print('Build raw dataframe test passed!')
```

Comment:
Remove the print statements. Testing frameworks will say which tests pass or fail so the test function name should be descriptive enough.

Reply:
Do this for all the print statements.